### PR TITLE
Theme The "Generate Overrides" Dialog

### LINKED ISSUE #61812: Scrolling Bars in Refactoring Popup Have Wrong Color
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Scrolling-Bars-in-Refactoring-Popup-Have/10067164)._

---
[regression] [worked-in:17.2.3]
Per the screenshot,  the refactoring popup is of white color, while its scrolling bars have black color.
The colors in the same window should match each other.

image
---
#### Feedback Bot on 6/10/2022, 01:04 AM: 

(private comment, text removed)

---
### Original Solutions
(no solutions)

### PR DESCRIPTION
**Pull request by ryzngard**
fixes #61812 

We were in a mix of themed and unthemed. This PR themes the entire dialog for use. 

### Original (Before Fix)

image

### Blue Theme 

image

### Dark Theme

image

### CODE REVIEW

**Review by Youssef1313 — CHANGES_REQUESTED** [review]
Change signature and add parameter have the same issue:

image

It would be great if they're fixed as well.

**Review by Youssef1313 — COMMENTED** [review]
Also extract interface:

image

and extract base class

image

File: src/VisualStudio/Core/Def/PickMembers/PickMembersDialog.xaml [hunk 1]
```diff
@@ -56,7 +63,8 @@
         <GroupBox x:Uid="MemberSelectionGroupBox"
                   Margin="0, 9, 0, 0"
                   Grid.Row="1"
-                  Header="{Binding ElementName=dialog, Path=PickMembersTitle}">
+                  Header="{Binding ElementName=dialog, Path=PickMembersTitle}"
+                  Foreground="{DynamicResource {x:Static vs:ThemedDialogColors.HeaderTextBrushKey}}">
```
Comment (JoeRobich) [inline src/VisualStudio/Core/Def/PickMembers/PickMembersDialog.xaml:67]: Is there a Theme color we should be using for the box Border?

Comment (ryzngard) [reply src/VisualStudio/Core/Def/PickMembers/PickMembersDialog.xaml:67]: It seems to be automatically done with `vs:ThemedDialogStyleLoader.UseDefaultThemedDialogStyles="True"`

**Review by CyrusNajmabadi — APPROVED** [review]

**Pull request merged**